.github/dependabot.yml

```diff
@@ -9,4 +9,4 @@ updates:
   - match:
       dependency_name: "io.quarkus:quarkus-universe-bom"
       # Quarkus 2 requires Java 11, which is too high.
-      version_requirement: ">=2.0.0.Final"
+      version_requirement: "2.x"
```

Comment:
it's worth trying.

https://dependabot.com/docs/config-file/#ignored_updates says

> The range format is specific to the package manager

So, we can try `[2.0.0.Final,)` as per [this issue](https://github.com/dependabot/dependabot-core/issues/1845) if "2.x" does not work.

Reply:
Nice find! As you mentioned, I will try "2.x" first and go with `[2.0.0.Final,)` if that doesn't work.